File: app/src/main/java/com/papi/nova/binding/PlatformBinding.java
package com.papi.nova.binding;

import android.content.Context;

import com.papi.nova.binding.audio.AndroidAudioRenderer;
import com.papi.nova.binding.crypto.AndroidCryptoProvider;
import com.papi.nova.nvstream.av.audio.AudioRenderer;
import com.papi.nova.nvstream.http.LimelightCryptoProvider;

public class PlatformBinding {
    public static LimelightCryptoProvider getCryptoProvider(Context c) {
        return new AndroidCryptoProvider(c);
    }
}

File: app/src/main/java/com/papi/nova/binding/PlatformBinding.kt
package com.papi.nova.binding

import android.content.Context
import com.papi.nova.binding.crypto.AndroidCryptoProvider
import com.papi.nova.nvstream.http.LimelightCryptoProvider

object PlatformBinding {
    @JvmStatic
    fun getCryptoProvider(c: Context): LimelightCryptoProvider = AndroidCryptoProvider(c)
}
